types/office-js-preview/index.d.ts

```diff
@@ -12487,24 +12497,34 @@ declare namespace Office {
         /**
          * Replaces the entire body with the specified text.
          *
-         * When working with HTML-formatted bodies, it is important to note that the `Body.getAsync` and `Body.setAsync` methods are not idempotent.
-         * The value returned from the `getAsync` method will not necessarily be exactly the same as the value that was passed in the `setAsync` method
-         * previously. The client may modify the value passed to `setAsync` in order to make it render efficiently with its rendering engine.
-         *
-         * When including links in HTML markup, you can disable online link preview by setting the `id` attribute on the anchor (\<a\>) to "LPNoLP"
-         * (see the **Examples** section for a sample).
-         *
-         * **Recommended**: Call `getTypeAsync` then pass the returned value to the `options.coercionType` parameter.
-         *
-         * **Important**: In Outlook on Windows and on Mac, the add-in user won't be able to revert this action with the **Undo** command.
-         *
          * @remarks
          * [Api set: Mailbox 1.3]
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
          *
+         * **Important**:
+         *
+         * - After the body is replaced with the specified content, the position of the cursor depends on which client the add-in is running.
+         * 
+         *   - **Windows**: The cursor appears at the beginning of the body of the mail item.
+         *
+         *   - **Mac**: The cursor position isn't preserved. The cursor disappears after the `setAsync` call and only reappears when the body of the mail item is selected.
+         *
+         *   - **Web**: The cursor appears at the end of the body of the mail item.
+         *
+         * - When working with HTML-formatted bodies, it's important to note that the `Body.getAsync` and `Body.setAsync` methods aren't idempotent.
```

Comment:
I don't think "idempotent" is the right word, and at any rate, let's try to describe the phenomenon without using a highly technical term that will be unknown to many readers.